pages/deploy/deploy_aws.md

```diff
@@ -223,41 +223,43 @@ have listed and defined all the parameters of the cdk.json file. If you still ha
+| vpc_id                                        | Optional                               | The VPC ID for the deployment account. If not provided, **a new VPC** will be created.                                                                                                                                                                                                                                                                                                                               |
+| vpc_endpoints_sg                              | Optional                               | The VPC endpoints security groups to be use by AWS services to connect to VPC endpoints. If not assigned, NAT outbound rule is used.                                                                                                                                                                                                                                                                                 |
+| vpc_restricted_nacl                           | Optional                               | If set to **true**, VPC NACLs added to restrict network traffic on the subnets of the data.all provisioned deployment VPC (default: false)                                                                                                                                                                                                                                                                           |
+| internet_facing                               | Optional                               | If set to **true**  CloudFront is used for hosting data.all UI and Docs and APIs are public. If false, ECS is used to host static sites and APIs are private. (default: true)                                                                                                                                                                                                                                        |
+| custom_domain                                 | Optional*                              | Custom domain configuration: `hosted_zone_name`, `hosted_zone_id`, `certificate_arn`, and `email_notification_sender_email_id`. If internet_facing parameter is **false** or `share_notifications.email` is active in `config.json` then custom_domain is REQUIRED for ECS ALB integration with ACM and HTTPS. It is optional when internet_facing is true.                                                          |
+| custom_domain                                 | [Strongly Recommended](#Enforce-TLS12) | Custom domain configuration for the frontend: `hosted_zone_name`, `hosted_zone_id`, `certificate_arn`, and `email_notification_sender_email_id`. If internet_facing parameter is **false** or `share_notifications.email` is active in `config.json` then custom_domain is REQUIRED for ECS ALB integration with ACM and HTTPS. It is optional when internet_facing is true.                                         |
```

Comment:
duplicate `custom_domain` - keep the Strongly Recommended one and to delete Optional

Reply:
ops good catch